docs/UserGuide.md

```diff
@@ -272,14 +293,26 @@ Format: `find com KEYWORD [ANOTHER_KEYWORD]...`
 - The order of the keywords does not matter. e.g. `Google Facebook` and `Facebook Google` will return the same results.
 - Only full words will be matched. e.g. `Googl` will not match `Google`.
 
+<div markdown="block" class="alert alert-info">
+
+:information_source: **Note:**<br>
+Upon successful execution of this command, your company list will be updated to only show matching results. All 
+subsequent delete, edit and view company commands will use indexing based on this updated list. Switching tabs will 
+not return the list to its full state. If you wish to return to seeing all the companies in your list, 
+execute [`list com`](#listing-all-companies-list-com).
+
+</div>
+
 Examples:
 - `find com Amazon`
 - Suppose you have this list of companies. Executing `find com google` will update the list to show matching
  companies: <br />
-     ![FindComGoogleSS](images/FindComGoogleSS.png) <br />
+     ![FindComGoogleSS](images/ug-company/FindComGoogleSS.png) <br />
+     <p align="center">Figure 7: Result of finding companies (example 1)</p>
      
      And executing `find com google facebook` will return the following: <br />
-     ![FindComGoogleFacebookSS](images/FindComGoogleFacebookSS.png) <br />
+     ![FindComGoogleFacebookSS](images/ug-company/FindComGoogleFacebookSS.png) <br />
+     <p align="center">Figure 8: Result of finding companies (example 2)</p>
 
 #### Listing all companies: `list com`
```

Comment:
I think we are still missing the pictures for delete and list right? Probably should put them in first before numbering the figures?

Reply:
My bad, I was not certain if we were adding picture for delete and list at first. I've added them in!